var/spack/repos/builtin/packages/rocclr/package.py

```diff
@@ -13,37 +13,67 @@ class Rocclr(CMakePackage):
        runtimes to work on Windows as well as on Linux without much effort."""
 
     homepage = "https://github.com/ROCm-Developer-Tools/ROCclr"
-    url      = "https://github.com/ROCm-Developer-Tools/ROCclr/archive/roc-3.5.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    def url_for_version(self, version):
+        # Fix up a typo in the 3.5.0 release.
+        if version == Version('3.5.0'):
+            return "https://github.com/ROCm-Developer-Tools/ROCclr/archive/roc-3.5.0.tar.gz"
+
+        url = "https://github.com/ROCm-Developer-Tools/ROCclr/archive/rocm-{0}.tar.gz"
+        return url.format(version)
+
+    version('3.7.0', sha256='a49f464bb2eab6317e87e3cc249aba3b2517a34fbdfe50175f0437f69a219adc')
     version('3.5.0', sha256='87c1ee9f02b8aa487b628c543f058198767c474cec3d21700596a73c028959e1')
 
     depends_on('cmake@3:', type='build')
-    for ver in ['3.5.0']:
+
+    for ver in ['3.5.0', '3.7.0']:
         depends_on('hsakmt-roct@' + ver, type='build', when='@' + ver)
         depends_on('hsa-rocr-dev@' + ver, type='build', when='@' + ver)
         depends_on('comgr@' + ver, type='build', when='@' + ver)
-        depends_on('mesa~llvm@18.3:', type='link', when='@' + ver)
 
+    depends_on('libelf', type='link', when='@3.7.0')
+    depends_on('numactl', type='link', when='@3.7.0')
+
+    depends_on('mesa~llvm@18.3:', type='link')
+
+    # Not yet merged: https://github.com/ROCm-Developer-Tools/ROCclr/pull/16
```

Comment:
So, it seems there was _some_ change making this patch redundant in 3.7.0, i'll have to update the comment